### PR TITLE
FeedbackResponseAttributes: rename giverEmail to giver and recipientEmail to recipient #5571

### LINKED ISSUE #5571: FeedbackResponseAttributes: rename giverEmail to giver and recipientEmail to recipient
Extension of issue:
Rename misleading instance variable names in FeedbackResponseAttributes #5456

### PR DESCRIPTION
**Pull request by sujeet14108**
Fixes #5571

### CODE REVIEW

**Comment by whipermr5** [comment]
@sujeet14108 Could you check where `toString` is used and report all the instances here?

**Comment by sujeet14108** [comment]
In `FeedbackResponsesDbTest`

        FeedbackResponseAttributes expected = getResponseAttributes("response1ForQ1S1C1");
        FeedbackResponseAttributes actual =
                frDb.getFeedbackResponse(expected.feedbackQuestionId, expected.giver, expected.recipient);

        assertEquals(expected.toString(), actual.toString());`

In `FeedbackResponsesDb`

     if (fr == null || JDOHelper.isDeleted(fr)) {

            throw new EntityDoesNotExistException(

                    ERROR_UPDATE_NON_EXISTENT + newAttributes.toString());
        }`

In `FeedbackSessionLogicTest`

       for (FeedbackResponseAttributes responsesForQn : actual.questionResponseBundle.get(expectedQuestion)) {
            actualResponses.add(responsesForQn.toString());
        }`
Similarly also at Line 631, 665 and 722.

In `FeedbackResponsesLogicTest`

     FeedbackResponseAttributes responseToUpdate = getResponseFromDatastore("response1ForQ2S1C1");

        responseToUpdate.responseMetaData = new Text("Updated Response");
        responseToUpdate.feedbackSessionName = "copy over";
        responseToUpdate.recipient = null;

        frLogic.updateFeedbackResponse(responseToUpdate);

        responseToUpdate = getResponseFromDatastore("response1ForQ2S1C1");
        responseToUpdate.responseMetaData = new Text("Updated Response");

        assertEquals(frLogic.getFeedbackResponse(responseToUpdate.feedbackQuestionId, responseToUpdate.giver,
                                                 responseToUpdate.recipient).toString(),
                     responseToUpdate.toString());`

Similarly also at Line 172, 182, and 199.

**Comment by whipermr5** [comment]
@sujeet14108 Good job! 👍

File: src/main/java/teammates/common/datatransfer/attributes/FeedbackResponseAttributes.java [hunk 1]
```diff
@@ -51,13 +51,13 @@ public FeedbackResponseAttributes() {
 
     public FeedbackResponseAttributes(String feedbackSessionName,
             String courseId, String feedbackQuestionId,
-            FeedbackQuestionType feedbackQuestionType, String giverEmail, String giverSection,
+            FeedbackQuestionType feedbackQuestionType, String giver, String giverSection,
             String recipientEmail, String recipientSection, Text responseMetaData) {
```
Comment (damithc) [inline src/main/java/teammates/common/datatransfer/attributes/FeedbackResponseAttributes.java]: Wait, should we change `recipientEmail` to `recipient` as well?

Comment (sujeet14108) [inline src/main/java/teammates/common/datatransfer/attributes/FeedbackResponseAttributes.java]: @damithc  Yes,  we should as recipientEmail was change to recipient in https://github.com/TEAMMATES/teammates/pull/5547. Some might have left.

Comment (sujeet14108) [inline src/main/java/teammates/common/datatransfer/attributes/FeedbackResponseAttributes.java]: So should i change this ?

Comment (whipermr5) [inline src/main/java/teammates/common/datatransfer/attributes/FeedbackResponseAttributes.java]: 👍 Go ahead

**Comment by sujeet14108** [comment]
Ready for review :)

**Pull request merged**